src/coreclr/inc/corhlprpriv.cpp

```diff
@@ -109,6 +109,27 @@ HRESULT _CountBytesOfOneArg(
     }
     switch (ulElementType)
     {
+        case ELEMENT_TYPE_GENERICINST:
+            // skip over generic type
+            CHECK_REMAINDER;
```

Comment:
I guess this is not strictly necessary in front of the `_CountBytesOfOneArg` calls, since the function ensures this is called before it reads anything from the signature.

Reply:
Actually I see the other cases do the same thing, so nevermind this.

Reply:
Sure, but looking at places where we call `_CountBytesOfOneArg` in here, it has a `CHECK_REMAINDER` before it.